flake.nix

```diff
@@ -1,7 +1,6 @@
 {
   inputs = {
     nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
-    flake-parts.url = "github:hercules-ci/flake-parts";
```

Comment:
This is actually needed as per documentation: https://github.com/hercules-ci/flake-parts/commit/14114c563acab4cee2828aa8dad9036d617e1ea2

Reply:
My bad, you had convinced me that it wasn't necessary in the past and everything worked without, so I thought it was fine, haha 🙂

I'll revert my commit soon, unless @frozolotl is faster than me 🙂